processccd.rst

```diff
@@ -1,131 +1,100 @@
-multiple packages in other astronomy analysis frameworks.
+-	``isr`` -  target=IsrTask - Task to perform instrumental signature removal or load a post-ISR image; the steps in ISR are to:
 
-The initial step is to correct the images for all the issues involved
-in taking a raw image CCD through to a processed one (e.g. doing the
-bias and dark current corrections, flat-fielding, etc.) by first doing
-everything that astronomers have used a medley of customized codes
-typically for each telescope before (like IRAF, UNIX shellscripts, IDL
-scripts etc.).  These tasks are usually grouped together under the
-general term 'Instrumental Signature Removal.'
+	- assemble raw amplifier images into an exposure with image, variance and mask planes
+	- perform bias subtraction, flat fielding, etc.
+	- mask known bad pixels
+	- provide a preliminary WCS
+		
+-	``charImage`` - target=CharacterizeImageTask - Task to characterize a science exposure, the steps of image characterization are to:
 
-The second step groups together several functions as 'Image
-Characterization', which includes for our purposes: object detection
-(very commonly done by Source Extractor), repairing of cosmic ray
-defects, measuring and subtracting of sky background, and then finally
-measuring bright sources and using this to estimate background and PSF
-of an exposure (which is often done currently by astronomers using the
-PSFex code).
-
-The last primary grouping of taks is what we will call 'Image
-Calibration', which measures faint sources, does the astrometry by
-fitting an improved WCS to the image (often done currently by
-astronomers using by using the SCAMP and SWARP codes, 'pinning' the
-image on the positions of known stars), and figures out the
-photometric zero-point for the image.
-
-
-How to use the ProcessCcd Task
-++++++++++++++++++++++++++++++
+	- detect sources, usually at high S/N
+	- estimate the background, which is subtracted from the image and returned as field "background"
+	- estimate a PSF model, which is added to the exposure
+	- interpolate over defects and cosmic rays, updating the image, variance and mask planes
+    
+ 
+-	``calibrate`` - target=CalibrateTask - Task to perform astrometric and photometric calibration, the steps are to:
 
-This task is primarily designed to be run from the command line.  The
-main method is 'run', which takes a single (butler) data reference for the
-raw input data (i.e. the directory in which all the raw data files are
-stored, and where all the processed data files are to be written).
+	- refine the WCS in the exposure
+	- refine the Calib photometric calibration object in the exposure
+	- detect sources, usually at low S/N
+ 
 
-..
- Preparing the data for ProcessCcd 
- ---------------------------------
+Entrypoint
+==========
 
+- ``lsst.pipe.tasks.processCcd.ProcessCcdTask.run`` 
+  
 
+Butler Inputs
+=============
 
+The main method, ``run``, takes a single butler data reference for the raw input data.
```

Comment:
"raw input data" -> What's the exact dataset name? Want to use that here and we'll link to its topic page.